spec/system/viewing_sidebar_spec.rb

```diff
@@ -5,7 +5,7 @@
   fab!(:user) { Fabricate(:user) }
   fab!(:category_sidebar_section_link) { Fabricate(:category_sidebar_section_link, user: user) }
 
-  before_all { sign_in(user) }
```

Comment:
This did not work for me locally for some reason even when running the whole spec file, we just use before everywhere else so I think it is fine here.

Also none of these specs pass locally even on `main`, maybe there is some site setting not right in my test db?

Reply:
O yea `before_all` doesn't work. Not sure why the tests pass previously.